Fabulous.XamarinForms/src/Fabulous.XamarinForms.Core/CustomControls.fs

```diff
@@ -15,96 +15,101 @@ open System.ComponentModel
 [<AllowNullLiteral>]
 type IViewElementHolder =
     inherit INotifyPropertyChanged
-    abstract ViewElement : ViewElement
+    abstract ViewElement : IViewElement
+    abstract ProgramDefinition : ProgramDefinition
 
 [<AllowNullLiteral>]
-type ViewElementHolder(viewElement: ViewElement) =
-    let ev = new Event<_,_>()
+type ViewElementHolder(viewElement: IViewElement, programDefinition: ProgramDefinition) =
+    let ev = Event<_,_>()
```

Comment:
You might also want to populate this event object on-demand (mutable, null initially)

Reply:
Thanks. Will take a look if that's possible. Not sure if Xamarin.Forms expects `PropertyChanged` right after the creation.